cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.master.startup;
+
+import co.cask.cdap.common.conf.CConfiguration;
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.common.guice.ConfigModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.junit.Test;
+
+/**
+ * Tests for {@link ConfigurationCheck}
+ * For now, it only tests Kafka topic validations. Later, tests for other configurations can be added.
+ */
+public class ConfigurationCheckTest {
+
+  @Test (expected = RuntimeException.class)
+  public void invalidLoggingKafkaTopicTest() {
+    CConfiguration cConf = CConfiguration.create();
```

Comment:
instead of this repetitive code, can you create a private method ``runConfigurationCheck(String param, String invalidValue)`` and call it with different values from each of these tests?